src/Service/PropertyExistenceChecker.php

```diff
@@ -11,12 +11,14 @@
 class PropertyExistenceChecker
 {
     private ReflectionClass  $reflectionClass;
-    private ?ReflectionClass $translationReflectionClass;
+    private ?ReflectionClass $translationReflectionClass = null;
 
-    public function __construct(object $entity)
+    public function __construct($entity)
```

Comment:
If now we are using string instead of object, please change type to `string`. And it will be good to change tests for this case (including tests for entity with constructor).